### DISCUSSION TITLE
Export 'Accordion' (imported as 'Accordion') was not found in '@vkontakte/vkui'

### DISCUSSION THREAD

**Discussion by Rezetchik** [VKCOM/VKUI#6070] [Q&A] [answered]
![Error](https://github.com/VKCOM/VKUI/assets/146764865/da89dfd9-6735-4096-830e-8aedfd5a77ee)

**Comment by inomdzhon** [comment]
Hi!

What kind of VKUI version you're using?

`Accordion` is exist since [v5.3.0](https://github.com/VKCOM/VKUI/releases/tag/v5.3.0).

**Reply by Rezetchik** [reply]
"@vkontakte/vkui": "5.1.0"
How can i update vkui?

**Reply by inomdzhon** [reply, marked as answer]
For **NPM**

```sh
npm install @vkontakte/vkui@latest
# OR (if you don't want update to latest)
npm install @vkontakte/vkui@5.3.0
```

For **Yarn**

```sh
yarn add @vkontakte/vkui@latest
# OR (if you don't want update to latest)
yarn add @vkontakte/vkui@5.3.0
```
